components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -183,10 +114,92 @@ export default class Workspaces extends React.Component<WorkspacesProps, Workspa
+                            </div>
+                            <input type="search" className="text-sm" placeholder="Search Workspaces" onChange={(v) => { if (workspaceModel) workspaceModel.setSearch(v.target.value) }} />
+                        </div>
+                        <div className="flex-1" />
+                        <div className="py-3">
+                        </div>
+                        <div className="py-3 pl-3">
+                            <DropDown prefix="Limit: " contextMenuWidth="w-32" activeEntry={workspaceModel ? workspaceModel?.limit + '' : undefined} entries={[{
+                                title: '50',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 50; }
+                            }, {
+                                title: '100',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 100; }
+                            }, {
+                                title: '200',
+                                onClick: () => { if (workspaceModel) workspaceModel.limit = 200; }
+                            }]} />
+                        </div>
+                        <button onClick={showStartWSModal} className="ml-2">New Workspace</button>
+                    </div>
+                    <ItemsList className="lg:px-28 px-10">
+                        <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                        {
+                            activeWorkspaces.map(e => {
+                                return <WorkspaceEntry key={e.workspace.id} desc={e} model={workspaceModel} stopWorkspace={wsId => getGitpodService().server.stopWorkspace(wsId)} />
+                            })
+                        }
+                        {
+                            activeWorkspaces.length > 0 && <div className="py-6"></div>
+                        }
+                        {
+                            inactiveWorkspaces.length === 0 ? null :
+                                <Item className="w-full bg-gray-50 py-3 px-3">
+                                    <ItemField className=" flex flex-col">
+                                        <div className="text-gray-400 text-sm text-center">Unpinned workspaces that have been inactive for more than 14 days will be automatically deleted. <a className="gp-link" href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection">Learn more</a></div>
+                                    </ItemField>
+                                </Item>
+                        }
+                        {
+                            inactiveWorkspaces.map(e => {
+                                return <WorkspaceEntry key={e.workspace.id} desc={e} model={workspaceModel} stopWorkspace={wsId => getGitpodService().server.stopWorkspace(wsId)} />
```

Comment:
q: can inactive workspaces be started to get stopped?

Reply:
not currently and I don't see how they woudl in future, but I think it would raise more questions passing an empty fucntion to bthe contract rather than a functioning one.